orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/api/manifests/kustomize/KustomizeBakeManifestRequest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Netflix, Inc.
```

Comment:
s/Netflix/Armory/

Reply:
GETS ME EVERY TIME!

Reply:
Haha, me too...there's a way to set at least IntelliJ to do this correctly but I've never learned how.